integration/sql/impl/Cargo.toml

```diff
@@ -13,4 +13,4 @@ crate-type = ["rlib"]
 
 [dependencies]
 anyhow = {workspace = true}
-sqlparser = {git = "https://github.com/mobuchowski/sqlparser-rs", branch = "sqlp-release"}
\ No newline at end of file
+sqlparser = {git = "https://github.com/mobuchowski/sqlparser-rs", branch = "snowflake/add-transient-keyword"}
```

Comment:
Is there a corresponding PR in `/
[sqlparser-rs](https://github.com/sqlparser-rs/sqlparser-rs)` ? 

What's the plan for future cooperation? Openlineage should not depend on personal forks and there is no guarantee the target PR won't change, not to mention the process of code reviews which does not exist

Reply:
>Is there a corresponding PR in / [sqlparser-rs](https://github.com/sqlparser-rs/sqlparser-rs) ?

It's already merged.

>What's the plan for future cooperation? Openlineage should not depend on personal forks and there is no guarantee the target PR won't change, not to mention the process of code reviews which does not exist

Then we should fork it in OL organization.

Reply:
> > Is there a corresponding PR in / [sqlparser-rs](https://github.com/sqlparser-rs/sqlparser-rs) ?
> 
> It's already merged.
> 
If it's merged, why do we point to fork? 
> > What's the plan for future cooperation? Openlineage should not depend on personal forks and there is no guarantee the target PR won't change, not to mention the process of code reviews which does not exist
> 
> Then we should fork it in OL organization.
It's a valid point.